src/pybind/mgr/dashboard/config_options.html

```diff
@@ -0,0 +1,114 @@
+
+{% extends "base.html" %}
+
+{% block content %}
+
+
+<script>
+  $(document).ready(function(){
+             // Pre-populated initial data at page load
+            var content_data = {{ content_data }};
+	    
+	     rivets.formatters.display_arrays = function(arr) {
+	         result = arr.join().replace(/,/g, "<br/>");
+	         return result;
+             };
+   
+             apply_filters = function() {
+                 content_data.options_list = [];
+                 var selection = "#" + content_data.service;
+                 $(selection).attr('selected','selected');
+                 var level = $("#level").val();
+                 var service = $("#service").val();
+                 if (level == "any" && service == "any") {
+                     content_data.options_list = content_data.options.options;
+                 }
+                 for (var opt of content_data.options.options) {
+                     if (service == "any" && opt.level == level) {
```

Comment:
When someone selects a level, they should also get all the lower levels (i.e. if I select "advanced" then it should show both advanced and basic options)